packages/cfd/src/Containers/cfd-password-modal/cfd-password-form/cfd-password-form.tsx

```diff
@@ -0,0 +1,289 @@
+import React from 'react';
+import { Formik, FormikErrors } from 'formik';
+import {
+    isDesktop,
+    getLegalEntityName,
+    getErrorMessages,
+    getCFDPlatformLabel,
+    CFD_PLATFORMS,
+    isMobile,
+} from '@deriv/shared';
+import { Text, FormSubmitButton, PasswordMeter, PasswordInput } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import {
+    TDxCompanies,
+    getDxCompanies,
+    getFormattedJurisdictionCode,
+} from '../../../Stores/Modules/CFD/Helpers/cfd-config';
+import { TCFDPasswordFormReusedProps, TCFDPasswordFormValues, TOnSubmitPassword } from '../cfd-password-modal';
+import PasswordStep from './password-step';
+
+type TCFDPasswordFormProps = TCFDPasswordFormReusedProps & {
+    account_title: string;
+    account_type: {
+        category?: string;
+        type?: string;
+    };
+    closeModal: () => void;
+    error_type?: string;
+    form_error?: string;
+    has_mt5_account: boolean;
+    is_bvi: boolean;
+    is_dxtrade_allowed: boolean;
+    is_real_financial_stp: boolean;
+    jurisdiction_selected_shortcode: string;
+    onCancel: () => void;
+    onForgotPassword: () => void;
+    should_set_trading_password: boolean;
+    show_eu_related_content: boolean;
+    submitPassword: TOnSubmitPassword;
+};
+
+const getAccountTitle = (
+    platform: string,
+    account_type: {
+        category?: string;
+        type?: string;
+    },
+    account_title: string
+) => {
+    if (platform === CFD_PLATFORMS.DXTRADE) {
+        return getDxCompanies()[account_type.category as keyof TDxCompanies][
+            account_type.type as keyof TDxCompanies['demo' | 'real']
+        ].short_title;
+    }
+
+    return account_title;
+};
+
+const handlePasswordInputChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    handleChange: (el: React.ChangeEvent<HTMLInputElement>) => void,
+    validateForm: (values?: TCFDPasswordFormValues) => Promise<FormikErrors<TCFDPasswordFormValues>>,
+    setFieldTouched: (field: string, isTouched?: boolean, shouldValidate?: boolean) => void
+) => {
+    handleChange(e);
+    validateForm().then(() => {
+        setFieldTouched('password', true);
+    });
+};
+
+const CFDPasswordForm = ({
+    account_title,
+    account_type,
+    closeModal,
+    error_message,
+    error_type,
+    form_error,
+    has_mt5_account,
+    is_real_financial_stp,
+    jurisdiction_selected_shortcode,
+    onCancel,
+    onForgotPassword,
+    platform,
+    should_set_trading_password,
+    show_eu_related_content,
+    submitPassword,
+    validatePassword,
+}: TCFDPasswordFormProps) => {
+    const button_label = React.useMemo(() => {
+        if (error_type === 'PasswordReset') {
+            return localize('Try later');
```

Comment:
@aizad-deriv It seems we should use `Localize` component instead of `localize`
@yashim-deriv can you explain the reason?